src/components/Radio/stories.js

```diff
@@ -1,46 +1,178 @@
 import React from "react";
+import { Form, Field } from "react-final-form";
 import { storiesOf } from "@storybook/react";
+import PropTypes from "prop-types";
+
+import Button from "../Button/index";
+import GridCol from "../GridCol/index";
+import GridRow from "../GridRow/index";
+import Layout from "../Layout/index";
+import MultiChoice from "../MultiChoice/index";
 import Radio from ".";
 
+const onSubmit = values => {
+  console.log(JSON.stringify(values, 0, 2));
+};
+
+const required = value => (value ? undefined : "Required");
+
+const RadioGroup = ({ label, hint, name, options, inline, ...input }) => (
+  <Layout>
+    <GridRow>
+      <GridCol>
+        <MultiChoice {...input} label={label} hint={hint}>
+          {options.map(o => (
+            <div key={o.value}>
+              <Radio
+                type="radio"
+                {...input}
```

Comment:
hmm feels weird to pass ...input to both the radio and the multichoice - why is this needed?